admin_requests.go

```diff
@@ -0,0 +1,41 @@
+package compute
+
+import (
+	"encoding/json"
+	"net/http"
+	"path"
+
+	"github.com/pkg/errors"
+)
+
+// EditorToken aquires and sets the editor token for the provided Runnable struct.
+// See: Runnable.Token()
+func (c *Client) EditorToken(runnable *Runnable) (*http.Response, error) {
+	// GET /api/v1/token/{environment}.{customerID}/{namespace}/{fnName}
+	req, err := c.adminRequestBuilder(http.MethodGet, path.Join("/api/v1/token", runnable.Path()), nil)
+	if err != nil {
+		return nil, err
+	}
+
+	res, err := c.httpClient.Do(req)
```

Comment:
Let's create a `client.do` function that wraps `httpClient.Do` and also returns an error if the response code is not 2XX, since that wouldn't cause an error normally.

Reply:
ok!